src/par_iter/test.rs

```diff
@@ -179,6 +179,57 @@ pub fn check_increment() {
 }
 
 #[test]
+pub fn check_skip() {
+    let a: Vec<usize> = (0..10).collect();
+
+    let mut b = vec![];
+    a.par_iter()
```

Comment:
@cuviper convinced me that we shouldn't call `weight_max()` -- but do make it longer :)